webpack.config.js

```diff
@@ -90,7 +90,8 @@ module.exports = function makeConfig(grunt) {
                 {
                     test: /\.jsx?$/,
                     loader: 'eslint-loader',
-                    exclude: shouldExclude
+                    // superdesk apps handle their own linter
+                    exclude: (p) => shouldExclude(p) || sdConfig.apps.some(app => p.indexOf(app) > -1),
```

Comment:
I find my solution more meaningful and less specific to npm, plus it works fine :)